test-it/testList.json

```diff
@@ -380,7 +382,6 @@
                 "TestVI_VectorScalarBinaryOp.via",
                 "TestVI_VectorUnaryOp.via",
                 "TestVI_VectorUnary2OutputOp.via",
-                "TestVI_VariantToData_VariantWiredToDataTypeTerminal.via",
```

Comment:
I think the test is worth keeping around just to ensure unwired inputs in hand written via error instead of crash the runtime (understanding that via generated by the editor will not have this case).

Reply:
Done.